api/pkg/apis/v1alpha1/managers/jobs/jobs-manager.go

```diff
@@ -524,7 +524,7 @@ func (s *JobsManager) HandleJobEvent(ctx context.Context, event v1alpha2.Event)
 				return err
 			}
 			var deployment model.DeploymentSpec
-			deployment, err = utils.CreateSymphonyDeploymentFromTarget(target, namespace)
+			deployment, err = utils.CreateSymphonyDeploymentFromTarget(ctx, target, namespace)
 			if err != nil {
```

Comment:
Add logs for error situation as I see no log in `Vendor` for the errors thrown.

Reply:
updated